include/Detectors/PetitDetector.hh

```diff
@@ -0,0 +1,216 @@
+#ifndef Petalo_hh
+#define Petalo_hh
+
+#include "VDetector.hh"
+using namespace std;
+
+//NOTES: best g1 for DD 0.1193, but for tritium 0.1146; S2 noise 1.9, 7.5%; g1_gas 0.1019, 0.1012
+//s2fano 3.6, 0.9; eField in gas 6.25, 6.2; e- life 650, 750 us; fid vol 80-130, 38-305 us; gasGap 4.25, 4.5 mm
+//DISCLAIMER: Slight differences from official published values due to private LUX algorithms
+
+class PetitDetector: public VDetector {
+
+public:
+
+  PetitDetector() {
+   cerr << "*** Detector definition message ***" << endl;
+   cerr << "You are currently using the Petalo detector." << endl << endl;
+
+    // Call the initialization of all the parameters
+    Initialization();
+  };
+  ~PetitDetector() override = default;
+
+  // Do here the initialization of all the parameters that are not varying as a function of time
+  void Initialization() override {
+
+    // Primary Scintillation (S1) parameters
+    g1 = 0.073;  // phd per S1 phot at dtCntr (not phe). Divide out 2-PE effect
+    sPEres = 0.58;  // single phe resolution (Gaussian assumed)
+    sPEthr = 0.35;  // POD threshold in phe, usually used IN PLACE of sPEeff
+    sPEeff = 1.00;  // actual efficiency, can be used in lieu of POD threshold
+    noiseBaseline[0] = 0.0;  // baseline noise mean in PE (Gaussian)
+    noiseBaseline[1] = 0.0;  // baseline noise width in PE (Gaussian)
+    noiseBaseline[2] = 0.0;  // baseline noise mean in e- (for grid wires)
+    noiseBaseline[3] = 0.0;  // baseline noise width in e- (for grid wires)
+    P_dphe = 0.2;  // chance 1 photon makes 2 phe instead of 1 in Hamamatsu PMT
+
+    coinWind = 100;  // S1 coincidence window in ns
+    coinLevel = 2;   // how many PMTs have to fire for an S1 to count
+    numPMTs = 89;    // For coincidence calculation
+
+    OldW13eV = true;  // for matching EXO-200's W measurement
+    // the "Linear noise" terms as defined in Dahl thesis and by Dan McK
+    noiseLinear[0] = 3e-2;  // S1->S1 Gaussian-smeared with noiseL[0]*S1
+    noiseLinear[1] = 3e-2;  // S2->S2 Gaussian-smeared with noiseL[1]*S2
+
+    // Ionization and Secondary Scintillation (S2) parameters
+    g1_gas = .0655;  // phd per S2 photon in gas, used to get SE size
+    s2Fano = 3.61;   // Fano-like fudge factor for SE width
+    s2_thr = 300.;  // the S2 threshold in phe or PE, *not* phd. Affects NR most
+    E_gas = 12.;    // field in kV/cm between liquid/gas border and anode
+    eLife_us = 2200.;  // the drift electron mean lifetime in micro-seconds
+
+    // Thermodynamic Properties
+    inGas = false;
+    T_Kelvin = 163.; // T of the value of density used in G4
```

Comment:
Let's use T = 164 and P = 1.26, which are the conditions of the real detector.

Reply:
You're right, I forgot!